extensions/reactive-mysql-client/runtime/src/main/java/io/quarkus/reactive/mysql/client/runtime/MySQLPoolRecorder.java

```diff
@@ -119,8 +124,12 @@ private MySQLConnectOptions toMySQLConnectOptions(DataSourceRuntimeConfig dataSo
         }
 
         if (dataSourceReactiveMySQLConfig.cachePreparedStatements.isPresent()) {
+            log.warn("cache-prepared-statements is not specific to a database kind, configure it at the parent level");
```

Comment:
If you include the key name in the message it's even more helpful.

Reply:
Done!